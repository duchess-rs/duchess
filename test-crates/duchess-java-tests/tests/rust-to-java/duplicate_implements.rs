duchess::java_package! {
    package log;

    public class log.Builder implements log.BuildStep, log.BuildStep {} //~ ERROR: duplicate reference
}

fn main() {}
