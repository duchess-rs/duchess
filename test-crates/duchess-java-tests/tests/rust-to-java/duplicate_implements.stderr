error: error in class `log.Builder`: duplicate reference in 'implements' to 'log.BuildStep'
 --> tests/ui/duplicate_implements.rs:4:5
  |
4 |     public class log.Builder implements log.BuildStep, log.BuildStep {}
  |     ^^^^^^

error: aborting due to 1 previous error

